packages/core/src/lib/toast/__tests__/toast.spec.ts

```diff
@@ -51,9 +51,7 @@ describe('toast', () => {
 
     const status = screen.getByRole('status');
     const toast = within(status).getByRole('listitem');
-    const actionButton = screen.getByRole('button', {
-      name: /perform action/iu,
-    });
+    const actionButton = screen.getByText(/action text/iu);
```

Comment:
`getByText` should usually be a last resort, or something that you use for finding `<p>` elements or other role-less entities. A `<button>` has a role of `button`, so this should continue to be a `getByRole`

```suggestion
    const actionButton = screen.getByRole('button', {
      name: /action text/iu,
    });
```